backend/app/database/database.py

```diff
@@ -57,5 +57,9 @@ def get_db_session():
     db = SessionLocal()
     try:
         yield db
+        db.commit()
```

Comment:
I am not sure we should do the commiting and the rollback here?
What is your reasoning behind adding this code? Isn't it safer to always explicitly commit where needed?

Reply:
Why do you need to write it explicitly everywhere if you can manage it in one place here?

The invocation always returns to line #59 when the request is over.

Reply:
For now I don't think we have a use case yet, but maybe not all db sessions want to commit at closing. You can leave for now and we'll see how it behaves